ui/component/common/form-components/form-field.jsx

```diff
@@ -261,7 +261,7 @@ export class FormField extends React.PureComponent<Props> {
                 {!noEmojis && openEmoteMenu && (
                   <Button
                     type="alt"
-                    className="button--file-action"
+                    className="button--file-action button--comment-icons"
```

Comment:
this should not need button--file-action
One class should be sufficient to specify what this should look like.
button--comment-icons should contain all that it needs.